apps/cowswap-frontend/src/modules/account/containers/Transaction/ActivityDetails.tsx

```diff
@@ -300,6 +329,23 @@ export function ActivityDetails(props: {
                   </>
                 )}
               </SummaryInnerRow>
+
+              {order && isCustomRecipient && order.receiver && (
+                <>
+                  <SummaryInnerRow>
+                    <b>Recipient:</b>
+                    <i>
+                      {order && (isCreating || isPending || isPresignaturePending) && isCustomRecipient && isWarningRecipientIconVisible && order.receiver && <Icon image={IconType.ALERT} color={UI.COLOR_ALERT} description="Alert" />}
```

Comment:
Same for warning icon: only show for `(isCreating || isPending || isPresignaturePending)` orders